.evergreen/orchestration/configs/servers/auth-aws.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "standalone-aws",
+  "auth_key": "secret",
+  "login": "bob",
+  "name": "mongod",
+  "password": "pwd123",
+  "procParams": {
+    "ipv6": true,
+    "bind_ip": "127.0.0.1,::1",
+    "logappend": true,
+    "journal": true,
+    "port": 27017,
+    "setParameter": {"enableTestCommands": 1, "authenticationMechanisms": "MONGODB-AWS,SCRAM-SHA-1"}
```

Comment:
Should this be `"MONGODB-AWS,SCRAM-SHA-256,SCRAM-SHA-1"`?

Reply:
It shouldn't hurt to add SCRAM-SHA-256. Added.

Reply:
I don't see SCRAM-SHA-256.